tests/Endpoint/AbstractSugarBeanEndpointTest.php

```diff
@@ -267,6 +267,23 @@ public function testGetRelated()
         $this->assertEquals('GET', self::$client->mockResponses->getLastRequest()->getMethod());
     }
 
+    /**
+     * @covers ::auditLog
+     */
+    public function testAuditLog()
+    {
+        $Bean = new Module();
+        self::$client->mockResponses->append(new Response(200));
+        //$Bean->setClient(self::$client);
+        $Bean->setUrlArgs(['Foo', 'bar']);
+        $Audit = $Bean->auditLog();
+        $this->assertInstanceOf('Sugarcrm\\REST\\Endpoint\\ModuleAudit', $Audit);
+        $Audit->fetch();
```

Comment:
you shouldn't have to call fetch here, instead it should be called in the method. Otherwise this test is wrong, as the API URL should be rest/v11_12/

Reply:
You can also test by explicitly setting the version on Client to < 11_12, and verifying that URL has updated version. You can also add JSON Data to the Response Object `{"records": [{...a fake audit record... }]}` , and verify that returned collection object has that record at(0)